lib/splash.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import './checkinternet.dart';
+import './webview.dart';
+
+class Splash extends StatefulWidget {
+  final bool debug;
+  const Splash({Key? key, required this.debug}) : super(key: key);
+
+  @override
+  State<Splash> createState() => _SplashState();
+}
+
+int checkInt = 0;
+
+class _SplashState extends State<Splash> {
+  @override
+  void initState() {
+    super.initState();
+    Future<int> internetState = CheckInternet().checkInternetConnection();
+    internetState.then((internetStatus) {
+      if (internetStatus == 0) {
+        setState(() {
+          checkInt = 0;
+        });
+        ScaffoldMessenger.of(context).showSnackBar(const SnackBar(
+          content: Text('No internet connection!'),
+        ));
+      } else {
+        setState(() {
+          checkInt = 1;
```

Comment:
How are we using checkInt?

Reply:
If there's no internet we show the users that there's no internet as a snack bar with screeners logo in the centre

Reply:
That we are anyway doing using snackbar. How are we using checkInt variable?

Reply:
Is checkInt variable required?

Reply:
I see we removed on the other PR.

Reply:
Let's remove here too and get it merged so that we resolve any conflicts on other PRs if any.